src/erldns_handler.erl

```diff
@@ -31,8 +41,8 @@ handle(dns:dname(), dns:type(), [dns:rr()], dns:message()) -> [dns:rr()].
 -include_lib("kernel/include/logger.hrl").
 -include_lib("dns_erlang/include/dns.hrl").
 
--define(MINIMUM_HANDLER_VERSION, 2).
--define(DEFAULT_HANDLER_VERSION, 3).
+-define(MIN_HANDLER_VER, 2).
+-define(DEF_HANDLER_VER, 3).
```

Comment:
While the renamed macros (MIN_HANDLER_VER and DEF_HANDLER_VER) are shorter, the original names (MINIMUM_HANDLER_VERSION, DEFAULT_HANDLER_VERSION) were more descriptive and self-explanatory. Unless there’s a strong reason (e.g., aligning with naming elsewhere or reducing verbosity in hot paths), I’d suggest keeping the original names for clarity.

Reply:
I just hate long lines, but this is really cosmetics :D